spine-godot/spine_godot/SpineCommon.h

```diff
@@ -56,6 +56,9 @@
 #define VARIANT_FLOAT Variant::REAL
 #define GDREGISTER_CLASS(x) ClassDB::register_class<x>()
 #define GEOMETRY2D Geometry
+#ifndef SNAME
+#define SNAME(m_arg) ([]() -> const StringName & { static StringName sname = _scs_create(m_arg); return sname; })()
```

Comment:
Need to check if _scs_create is actually exposed in 3.x.

Reply:
I think it is???  I 'll have to check, we are using a custom 3.x fork...